src/mapInteractor.js

```diff
@@ -64,28 +55,82 @@ var mapInteractor = function (args) {
     {
       throttle: 30,
       discreteZoom: false,
-      panMoveButton: 'left',
-      panMoveModifiers: {},
-      zoomMoveButton: 'right',
-      zoomMoveModifiers: {},
-      rotateMoveButton: 'left',
-      rotateMoveModifiers: {ctrl: true},
-      panWheelEnabled: false,
-      panWheelModifiers: {},
-      zoomWheelEnabled: true,
-      zoomWheelModifiers: {},
-      rotateWheelEnabled: true,
-      rotateWheelModifiers: {ctrl: true},
+
+      /* There should only be one action with any specific combination of event
+       * and modifiers.  When that event and modifiers occur, the specified
+       * action is triggered.  The event and modifiers fields can either be a
+       * simple string or an object with multiple entries with each entry set
+       * to true, false, or undefined.  If an object, all values that are
+       * truthy must match, all values that are false must not match, and all
+       * other values that are falsy are ignored.
+       *   Available actions:
+       * pan, zoom, rotate, select, zoomselect, unzoomselect, click
+       *   Available events:
+       * left, right, middle, wheel
+       *   Available modifiers:
+       * shift, ctrl, alt, meta
+       *   Useful fields:
+       * action: the name of the action.  Multiple events may trigger the same
+       *    action.
+       * event: the name of the event or an object with event names for keys
+       *    and boolean values that indicates the combination of events that
+       *    trigger this action.
+       * modifiers: the name of a modifier or an object with modifier names for
+       *    keys and boolean values that indicates the combination of modifiers
+       *    that trigger this action.
+       * selectionRectangle: true if a selection rectangle should be shown
+       *    during the action.
+       * name: a string that can be used to reference this action.
+       * owner: a string that can be used to reference this action.
+       */
+      actions: [{
+        action: 'pan',
```

Comment:
Actions don't necessarily correspond to events (though currently they all do), so to have constants for them I think it would be more proper to make another source file and define geo_action.pan = 'pan', etc.  What do you think?